full-stack-practice/client/src/Catalog.tsx

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useState } from 'react';
+import { Product } from '../../server/server.ts';
+import { toDollars } from '../src/lib/index';
```

Comment:
In React you can simplify this import to just `from './lib';`.

You should never have an import statement that includes `src`. (You went up one level too many.)